### PR TITLE
Finishes Display class.

### PR DESCRIPTION
**Pull request by tomsons26**

### CODE REVIEW

File: src/game/client/display.cpp [hunk 1]
```diff
@@ -74,9 +78,110 @@ void Display::Reset()
  */
 void Display::Update()
```
Comment (tomsons26) [inline src/game/client/display.cpp:79]: This function needs a checkup by someone else, it seems to work right but im not sure if i didn't miss anything

File: src/game/client/display.cpp [hunk 2]
```diff
@@ -74,9 +78,110 @@ void Display::Reset()
...
+        } else if (m_someLogoMovieInt2 < 0 && m_someLogoMovieInt1 < 0) {
+            Stop_Movie();
+        } else {
+            // some sort of text timing, we never see the text cause its color is black
+            if (!m_unkInt) {
+                if (m_unkDisplayString != nullptr) {
+                    Delete_Instance(m_unkDisplayString);
+                }
+
+                m_unkDisplayString = g_theDisplayStringManager->New_Display_String();
```
Comment (OmniBlade) [inline src/game/client/display.cpp:96]: Based on this the unknown string is something like `m_copyrightString` or some such?

File: src/game/client/display.cpp [hunk 3]
```diff
@@ -74,9 +78,110 @@ void Display::Reset()
  */
 void Display::Update()
 {
-#ifdef GAME_DLL
-    Call_Method<void, Display>(PICK_ADDRESS(0x00421870, 0x007BDB67), this);
-#endif
+    /*
```
Comment (OmniBlade) [inline src/game/client/display.cpp]: No need to leave this comment block here, I'll check the function over before I give it final approval.

**Review by OmniBlade — CHANGES_REQUESTED** [review]

File: src/game/client/display.cpp (hunk 2 above)
Comment (tomsons26) [reply src/game/client/display.cpp:96]: To my best ability to tell this is a general purpose slot used in sort of emergency cases(in that this is not the intended way to handle this at all based on code i've seen so far).
W3DDisplay::Draw for instance, seems would use it for some debug prints, not clear what.

**Review by OmniBlade — APPROVED** [review]

**Pull request merged**